utils/build/docker/nodejs/parametric/server.js

```diff
@@ -371,6 +374,52 @@ app.post("/trace/otel/otel_set_baggage", (req, res) => {
   res.json({ value });
 });
 
+app.post('/ffe/start', async (req, res) => {
```

Comment:
Can we update the test_parametric_endpoints tests to validate this new endpoint. We should add a very simple test that validates either the core functionality or asserts that the new endpoint returns a successful status code.